lesson-5/index.css

```diff
@@ -153,7 +153,7 @@ body{
     width:35%;
     text-align: center;
 }
-.main__second button{
+.main__second--button{
```

Comment:
Good.
But you can use here `.main__second-button` name.
It will be an element of `main` block